docs/chains/Introduction.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 0
+title: Introduction
+---
+
+Darwinia offers a number of different chains for development of messaging infrastructure and Dapp deployment, each with its own features and benefits.
+
+## [Darwinia Chain](darwinia-chain.md)
+
+Darwinia Chain(DC) is designed to be the primary chain for tokens and governance, main communication hub and routing point of the LCMP, and will support most on-chain light clients for other public chains.
```

Comment:
Should we remove the short forms? We almost never use it.

Reply:
Just to confirm, do the short forms above mean `DC` or `DP`?

Reply:
DC for Darwinia Chain